.cloud-repo-tools.json

```diff
@@ -0,0 +1,13 @@
+{
+  "_product": "tasks",
+  "id": "tasks",
+  "name": "TBD",
```

Comment:
There should be no TBDs. Please either edit the TBDs, or (better) make a PR to [repo-tools](https://github.com/GoogleCloudPlatform/nodejs-repo-tools) and add this API (then you won't need to override `_product`).

Reply:
I'm not sure what this json does. @alexander-fenster